restservice/src/main/java/org/itevents/service/transactional/MyBatisEventService.java

```diff
@@ -102,7 +97,6 @@ public Event getFutureEvent(int eventId) {
         Event event = getEvent(eventId);
         if (event.getEventDate().before(new Date())) {
             String message = String.format("Try to get event id=%s with date %s as future event", eventId, event.getEventDate().toString());
```

Comment:
@IgorMaksymov I hink that you have to refactor this message to somehing like this: "Event with id=%s is passed"